Provided by David Malcolm <dmalcolm@redhat.com> to address a build
failure with clang 9.0.1.

--- UTC
Index: gcc/analyzer/engine.cc
===================================================================
--- gcc/analyzer/engine.cc
+++ gcc/analyzer/engine.cc
@@ -2940,7 +2940,7 @@ template <>
 inline void
 pod_hash_traits<function_call_string>::mark_empty (value_type &v)
 {
-  v.m_fun = reinterpret_cast<function *> (NULL);
+  v.m_fun = static_cast<function *> (NULL);
 }
 template <>
 inline bool
@@ -2952,7 +2952,7 @@ template <>
 inline bool
 pod_hash_traits<function_call_string>::is_empty (value_type v)
 {
-  return v.m_fun == reinterpret_cast<function *> (NULL);
+  return v.m_fun == static_cast<function *> (NULL);
 }
 
 namespace ana {
